jsx/Filter.js

```diff
@@ -1,4 +1,4 @@
-import React, {Component} from 'react';
+import React, {useEffect} from 'react';
```

Comment:
MAJOR collisions in this file mainly between the commits referenced below
main: 2372c4294cdfcdcce2843bbf6eb7a2d2df20a640
23: b52a8a9592041af2040fd9d121642619f53d9146

Reply:
@HenriRabalais looks like you're the author of both of the above commits, can you double check this file?

Reply:
@driusan done!